google-cloud-logging-bom/pom.xml

```diff
@@ -50,21 +50,20 @@
 
   <dependencyManagement>
     <dependencies>
-
       <dependency>
-        <groupId>com.google.api.grpc</groupId>
-        <artifactId>proto-google-cloud-logging-v2</artifactId>
-        <version>0.90.2-SNAPSHOT</version><!-- {x-version-update:proto-google-cloud-logging-v2:current} -->
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-logging</artifactId>
+        <version>3.1.2-SNAPSHOT</version><!-- {x-version-update:google-cloud-logging:current} -->
       </dependency>
       <dependency>
         <groupId>com.google.api.grpc</groupId>
         <artifactId>grpc-google-cloud-logging-v2</artifactId>
         <version>0.90.2-SNAPSHOT</version><!-- {x-version-update:grpc-google-cloud-logging-v2:current} -->
       </dependency>
       <dependency>
-        <groupId>com.google.cloud</groupId>
-        <artifactId>google-cloud-logging</artifactId>
-        <version>3.1.2-SNAPSHOT</version><!-- {x-version-update:google-cloud-logging:current} -->
+        <groupId>com.google.api.grpc</groupId>
+        <artifactId>proto-google-cloud-logging-v2</artifactId>
+        <version>0.90.2-SNAPSHOT</version><!-- {x-version-update:proto-google-cloud-logging-v2:current} -->
```

Comment:
is there any specific reason to change order of the dependencies?

Reply:
The change in the order of dependencies doesn't make any difference since maven creates its own dependency tree and resolves the order. Owlbot  reorders the dependencies purely for consistency across libraries.